skatepark/sass/blocks/_post-comments.scss

```diff
@@ -0,0 +1,169 @@
+@import "../../../blockbase/sass/blocks/button-mixins";
+
+.wp-block-post-comments {
+
+	line-height: var(--wp--custom--body--typography--line-height);
+
+	.reply {
+		margin-bottom: var(--wp--custom--margin--vertical);
+
+		a {
+			--wp--custom--button--typography--font-size: var(--wp--preset--font-size--normal);
+			@include button-border-styles;
+			--wp--custom--button--color--text: var(--wp--custom--button--hover--color--text);
+			--wp--custom--button--color--background: var(--wp--custom--button--color--background);
+			--wp--custom--button--border--color: var(--wp--custom--button--hover--color--text);
+
+			& ,
+			&:hover,
+			&:focus,
+			&:active {
+				line-height: 1;
+			}
+			&:not(.has-background):not(.has-text-color) {
+				&:hover,
+				&:focus,
+				&.has-focus {
+					--wp--custom--button--typography--font-size: var(--wp--preset--font-size--normal);
+					--wp--custom--button--color--text: var(--wp--custom--button--hover--color--background);
+					--wp--custom--button--color--background: var(--wp--custom--button--hover--color--text);
+				}
+			}
+		}
+	}
+
+	.comment-reply-title {
+		margin-bottom: 30px;
+	}
+
+	form {
+		display: grid;
+		column-gap: 1.2em;
+		grid-template-rows: auto;
+		grid-template-areas:
+			"notes notes"
+			"comment comment"
+			"author email"
+			"url url"
+			"cookies-consent cookies-consent"
+			"form-submit form-submit";
+
+		input:not([type=submit]):not([type=checkbox]),
+		textarea {
+			border: 3px solid var(--wp--custom--color--primary);
+			background: var(--wp--custom--color--background);
+			font-size: var(--wp--preset--font-size--normal);
+			width: 100%;
+		}
+
+		label:not([for=wp-comment-cookies-consent]) {
+			font-size: var(--wp--preset--font-size--tiny);
+			font-weight: 900;
+			margin-bottom: var(--wp--custom--margin--baseline);
+			text-transform: uppercase;
+		}
+
+		p {
+			margin-top: 0;
+			margin-bottom: 30px;
+		}
+
+		.comment-notes {
+			grid-area: notes;
+		}
+
+		.comment-form-author {
+			grid-area: author;
+		}
+
+		.comment-form-email {
+			grid-area: email;
+		}
+
+		.comment-form-url {
+			grid-area: url;
+		}
+
+		.comment-form-comment {
+			grid-area: comment;
+
+			textarea {
+				vertical-align: top;
+			}
+		}
+
+		.comment-form-cookies-consent {
+			grid-area: cookies-consent;
+
+			input[type="checkbox"] {
+				@include hide-accesibly;
+				& + label {
+					margin-left: 0;
+					padding-left: 3em;
+					position: relative;
+				}
+				& + ::before,
+				& + ::after {
+					box-sizing: border-box;
+				}
+				& + ::before {
+					content: "";
+					position: absolute;
+					top: -0.4em;
+					left: 0;
+					width: 30px;
+					height: 30px;
+					border: 3px solid var(--wp--custom--color--primary);
+				}
```

Comment:
I see this is also on Quadrat, I'd say this should be on Blockbase too, but I'd appreciate other team member's opinions. Ideally we should add a checkbox inside theme.json's `custom.form` section. I feel strongly that we need consistent form styles defined for the theme from theme.json and reused whenever needed. This will be helpful when we style things like WooCommerce's forms, for example. A mixin for checkboxes like we have for buttons could probably be useful if I'm thinking ahead.

Reply:
I agree, this should be moved to Blockbase.